multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/persistence/dto/TextAttributeConverterTest.java

```diff
@@ -0,0 +1,41 @@
+package org.cloudfoundry.multiapps.controller.core.persistence.dto;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class TextAttributeConverterTest {
+
+    private final TextAttributeConverter textAttributeConverter = new TextAttributeConverter();
+
+    @Test
+    void testConvertToEntityAttribute() {
+        Assertions.assertEquals("test", textAttributeConverter.convertToEntityAttribute("test"));
+    }
+
+    @Test
+    void testConvertToDatabaseColumnWhenInputIsNotNull() {
+        Assertions.assertEquals("test", textAttributeConverter.convertToDatabaseColumn("test"));
+    }
+
+    @Test
+    void testConvertToDatabaseColumnWhenInputIsNull() {
+        Assertions.assertEquals("", textAttributeConverter.convertToDatabaseColumn(null));
+    }
+
+    @Test
+    void testConvertToDatabaseColumnWhenInputIsLongerThan4000chars() {
+        String expectedString = getLongStringInput().substring(0, 3997)
+                                                    .concat("...");
+
+        Assertions.assertEquals(expectedString, textAttributeConverter.convertToDatabaseColumn(getLongStringInput()));
+    }
+
+    private String getLongStringInput() {
+        int stringLength = 5000;
+        StringBuilder longStringBuilder = new StringBuilder();
+        for (int i = 1; i <= stringLength; i++) {
+            longStringBuilder.append("a");
+        }
+        return longStringBuilder.toString();
+    }
```

Comment:
This long string generation is fine, but if you like, you may look at this examples: https://stackoverflow.com/questions/2804827/create-a-string-with-n-characters
There are some cool one-liners.